src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
```

Comment:
```suggestion
  return {
```
just FYI - if you don't do anything else after creating this object but just return it - you may do it immediately